state/state.go

```diff
@@ -17,6 +17,10 @@ import (
 const newL2BlockEventBufferSize = 500
 
 var (
+	// DefaultSenderAddress is the address that jRPC will use
+	// to communicate with the state for eth_EstimateGas and eth_Call when
+	// the From field is not specified because it is optional
+	DefaultSenderAddress = "0x1111111111111111111111111111111111111111"
```

Comment:
Maybe we can set here what geth does and use the `ZeroAddress`.
Then, if the `from` is unspecified, we set the from to `ZeroAddress`.